config.example/group_vars/slurm-cluster.yml

```diff
@@ -3,7 +3,7 @@
 ################################################################################
 # Slurm job scheduler configuration
 # Playbook: slurm, slurm-cluster, slurm-perf, slurm-perf-cluster, slurm-validation
-slurm_version: 20.11.3
+slurm_version: 20.11.6
```

Comment:
This should be 20.11.7. Not just because it's the latest, but because SchedMD appears to have removed all older revisions of 20.11.x from their download site...

https://download.schedmd.com/slurm/

![image](https://user-images.githubusercontent.com/279339/118051267-ef0e4980-b33d-11eb-8341-e6faec3dc542.png)

Reply:
That's a little concerning, when I updated a few days ago the latest version was 20.11.6. I wonder if they had another security issue